data-prepper-test-common/src/main/java/org/opensearch/dataprepper/matcher/MapEquals.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.dataprepper.matcher;
```

Comment:
I think it would be good to use the package `org.opensearch.dataprepper.test.matcher`. Then any future test packages could all go under `org.opensearch.dataprepper.test`.

Thoughts?

Reply:
I think it'll be redundant since the project name already has test. If not I can change it to `org.opensearch.dataprepper.test.matcher`

Reply:
This does follow the convention we have in some other areas of Data Prepper though. For example, the project name for the logstash converter is `data-prepper-logstash-configuration-converter`, and the package for the classes in this project start with `org.opensearch.dataprepper.logstash`.

Reply:
I think what David meant is that should the package naming be `org.opensearch.dataprepper.matcher` or `org.opensearch.dataprepper.test.matcher`
And there's an open issue about `org.opensearch.dataprepper` package naming for 2.0 release
https://github.com/opensearch-project/data-prepper/issues/344

Reply:
This is a Java package name, not the project name. They are different concepts.

While they are different concepts, many projects use similar naming within the project and the package to help track where something came from. Seeing `org.opensearch.dataprepper.test...` is a clear indication that this is part of the testing project.

Regarding #344, thank you for using `org.opensearch` here! Since this is a new project it will work. In 2.0 all the `com.amazon` is going to have to go away.